demisto_sdk/commands/validate/file_validator.py

```diff
@@ -720,7 +720,10 @@ def is_valid_structure(self):
         if self.use_git:
             if self.branch_name != 'master' and (not self.branch_name.startswith('19.') and
                                                  not self.branch_name.startswith('20.')):
-                print('Validates only committed files')
+                if not self.is_circle:
```

Comment:
Shouldn't this be checking the `post_commit` option?

Reply:
For some weird reason `post_commit` is `is_circle`